webapp-admin/package.json

```diff
@@ -28,5 +28,17 @@
     "node-sass": "^4.12.0",
     "react-dom": "^16.12.0",
     "vanilla-framework": "^1.7.1"
+  },
+  "browserslist": {
```

Comment:
this was automatically added by running `npm run build` so I assume it's OK here